content/blog/2019-10-21-rmangal.md

```diff
@@ -0,0 +1,154 @@
+---
+slug: rmangal
+title: 'rmangal: making ecological networks easily accessible'
+date: 2019-10-21
+authors:
+  - Kevin Cazelles
+  - Steve Vissault
+categories: blog
+topicid: 1848
+tags:
+  - Software Peer Review
+  - R
+  - packages
+  - open-data
+  - data
+  - data-access
+  - ecology
+  - ecological networks
+  - API
+  - API client
+  - Mangal
+  - rmangal
+---
+
+
+In early September, the version 2.0.0 of [rmangal][rmangal] was approved by
+rOpenSci, four weeks latter it made the CRAN. Following-up on our experience we
+detail below the reasons why we wrote rmangal, why we submitted our package to
+rOpenSci and how the peer-review improve our package.
+
+
+
+### [Mangal](https://mangal.io/#/), a database for ecological networks
```

Comment:
Best not to link Mangal in heading since it doesn't look nice

Reply:
:+1:

Reply:
Done